java/src/test/java/com/mabl/example/selenium/UseCalculatorLocalTest.java

```diff
@@ -0,0 +1,29 @@
+package com.mabl.example.selenium;
+
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.JavascriptExecutor;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class UseCalculatorLocalTest extends SeleniumTestWithProxySupport {
+
+    public UseCalculatorLocalTest() {
+        super(false);
+    }
+
+    @Test
+    public void useCalculator() throws Exception {
+        driver.get("http://juliemr.github.io/protractor-demo/");
```

Comment:
Do you think there is any value is using something under our control? Depending on the license we could just fork this or copy it into the test lab :shrug:

Reply:
Yes, good point, and actually I think we _do_ have something very similar somewhere.  A calculator app hosted out of some GCS bucket.  Let me see if I can find that URL.

Reply:
lol I spent forever trying to find this thinking it was some obscure GCS bucket.  Turns out it's just `http://calc.mabl.com/`

Reply:
Unfortunately our calculator is completely button-based.  Going to have to stick with the protractor demo for now because I want a mixture of keyboard input and button clicks.  Maybe at some point we can clone that and host a copy ourselves.